tests/Unit/Core/Form/IdentifiableObject/DataProvider/ProductFormDataProviderTest.php

```diff
@@ -1546,6 +1590,9 @@ private function getDefaultOutputData(): array
      */
     private function buildProvider(CommandBusInterface $queryBusMock, bool $activation): ProductFormDataProvider
     {
+        $urlGeneratorMock = $this->getMockBuilder(UrlGeneratorInterface::class)->getMock();
+        $urlGeneratorMock->method('generate')->willReturnArgument(0);
```

Comment:
This part is not used anymore and needs to be removed (along with the associated use statement)